windows/src/engine/keyman32/k32_lowlevelkeyboardhook.cpp

```diff
@@ -222,10 +222,12 @@ LRESULT _kmnLowLevelKeyboardProc(
 
   */
 
-  if (hs->dwExtraInfo != 0 || hs->scanCode == SCAN_FLAG_KEYMAN_KEY_EVENT || hs->vkCode == VK_PROCESSKEY || hs->vkCode == VK_PACKET) {
+  if (hs->dwExtraInfo != 0 || hs->scanCode == SCAN_FLAG_KEYMAN_KEY_EVENT ||
+      hs->vkCode == VK_PROCESSKEY || hs->vkCode == VK_PACKET || !isKeymanKeyboardActive) {
```

Comment:
```suggestion
  if (hs->dwExtraInfo != 0 || 
      hs->scanCode == SCAN_FLAG_KEYMAN_KEY_EVENT ||
      hs->vkCode == VK_PROCESSKEY || 
      hs->vkCode == VK_PACKET || 
      !isKeymanKeyboardActive) {
```

Easier to spot that they are all separate conditions if we split across multiple lines